File-Formats/PDF/Convert-HTML-To-PDF/WPF.md

```diff
@@ -10,15 +10,15 @@ keywords: Assemblies
 # Convert HTML to PDF file in WPF
 ## Steps to convert Html to PDF document in WPF:
 
-Create a new WPF application project.
+1. Create a new WPF application project.
 ![Convert HTMLToPDF WPF Step1](htmlconversion_images/WPF_Step1.png)
 Now, the project configuration window appears. Click Create button to create a new project with the default project configuration.
 ![Convert HTMLToPDF WPF Step2](htmlconversion_images/WPF_Step2.png)
 
-Install the [Syncfusion.HtmlToPdfConverter.WPF](https://www.nuget.org/packages/Syncfusion.HtmlToPdfConverter.WPF) NuGet package as a reference to your WPF application [NuGet.org](https://www.nuget.org/)
+2. Install the [Syncfusion.HtmlToPdfConverter.WPF](https://www.nuget.org/packages/Syncfusion.HtmlToPdfConverter.WPF) NuGet package as a reference to your WPF application [NuGet.org](https://www.nuget.org/)
 ![Convert HTMLToPDF Docker Step3](htmlconversion_images/WPF_Step3.png)
 
-Include the following namespaces in the MainWindow.xaml.cs file.
+3. Include the following namespaces in the MainWindow.xaml.cs file.
```

Comment:
Continue the numbering for create button and code

Reply:
Added the numbering for create button and code